docs/university.md

```diff
@@ -2540,6 +2540,16 @@ jsjxy.hbut.edu.cn 证书链不全，自建 RSSHub 可设置环境变量 NODE_TLS
 
 <Route author="NeverBehave" example="/slu/tyyjkxy/14754" path="/slu/tyyjkxy/:id" :paramsDesc="['类别ID']" />
 
+## 上海社会科学院
+
+### 研究生院
+
+<Route author="yanbot-team" example="/sassorgcn" path="/sassorgcn/gs/:type" :paramsDesc="['类别ID，其他未列出的栏目参数可以从页面的 URL Path 中找到，例如：信息公告的网址为https://gs.sass.org.cn/1799/list.htm，则类别ID为1799']" radar="1" >
```

Comment:
Please use path `/sass/*`

Reply:
> Please use path `/sass/*`

`/sass/*` has different meanings and I'm afraid it might cause misunderstanding。  
In this route, sass stands for Shanghai Academy of Social Sciences，but  `/sass/*`also stands for [a CSS extension language](https://www.sass.hk/).